src/python/dxpy/nextflow/nextflow_utils.py

```diff
@@ -85,11 +94,14 @@ def write_dxapp(folder, content):
         json.dump(content, dxapp)
 
 
-def get_regional_options(region, resources_dir):
+def get_regional_options(region, resources_dir, cache_docker):
     nextaur_asset, nextflow_asset = get_nextflow_assets(region)
     regional_instance_type = get_instance_type(region)
-    image_refs = None
-    image_bundled = None
+    if cache_docker:
+        image_refs = run_nextaur_collect(resources_dir)
+        image_bundled = bundle_docker_images(image_refs)
+    else:
+        image_bundled = {}
```

Comment:
should be an empty array

Reply:
[8fa5344](https://github.com/dnanexus/dx-toolkit/pull/1280/commits/8fa5344430a72b77a305f2d2f72f49b2ea7859e4)